### ISSUE TITLE
More exactly defining dot size and spacing (like ttaudio)

### ISSUE THREAD

**Issue by SciLor** [entropia/tip-toi-reveng#183]
It would be nice if I could define the exact size of the dot / spacing I want to print in µm like in ttaudio, including dpi.
My printer works fine with ttaudio around the standard sized/spacings. But I was unable to print any oid-codes with tttool.

**Comment by nomeata** [comment]
Are you printing PDF/SVG/PNGs? What is the difference between ttaudio’s and tttools’ default settings?

**Comment by SciLor** [comment]
I want to create pngs because I reference them in photoshop (smart objects). So I can replace same easily.
In ttaudio you define the size and the spacing of the oid-codes in µm. I havn't found anything similar in tttools.
Just take a look at the ttaudio test sheet.

**Comment by nomeata** [comment]
Do you know about the options `--dpi` and `--pixel-size`? Can you recreate `tttaudio`’s settings using them?

Hmm, the idea of a test page with many different settings on one page is a good one, maybe tttool should offer the same.

**Comment by SciLor** [comment]
Yes, I know those options. But ttaudio scales in µm. If you take 600 dpi you have around 42µm pixel size. (This would be pixel-size = 1). Mostly any value between 41µm - 60µm (0.999-1.5 in pixel size) works fine, combined with a spacing of 103µm - 166µm (I don't know what is the counter-value in tttools).

**Comment by nomeata** [comment]
Going through old issues. I am still not quite sure what settings are needed here…

If you can control dpi and pixel-size, doesn’t this completely determine the layout (the distance between the dots is predetermined by the OID standard, isn’t it?)

**Comment by SciLor** [comment]
I really don't remember exactly. I think it would have been useful to set the sizes in µm like in the ttaudio. Iirc the png dimension should not change in my case. This would have been problematic for the smart objects.
